integration_test/Bindings/Python/dialects/hw.py

```diff
@@ -83,6 +83,14 @@ def build(module):
   # CHECK: #hw.param.decl<"param2": i32>
   print(pdecl)
 
+  # CHECK: #hw.param.decl.ref<"param2"> : i32
+  pdeclref = hw.ParamDeclRefAttr.get(ctx, "param2")
+  print(pdeclref)
+
+  # CHECK: !hw.int<#hw.param.decl.ref<"param2">>
+  pinttype = hw.IntType.get_from_param(ctx, "param2")
```

Comment:
We want to expose the lowest level possible in the bindings. So `get_from_param` should accept an `Attribute` which should be `pdeclref` in this case.